org.musicbrainz.Picard.json

```diff
@@ -208,7 +189,7 @@
             "name": "picard",
             "buildsystem": "simple",
             "build-commands": [
-                "${PIP} install --egg --no-deps --prefix=/app --install-option='--disable-autoupdate' ."
+                "${PYTHON} setup.py install --prefix=/app --disable-autoupdate"
```

Comment:
Does the pip command not work any more?

If it does, I'd prefer we keep it, as it's more future-proof.

Reply:
The pip command does not work anymore since they removed the `--egg` option. From what I saw on the corresponding PR, they recommended this way instead.

Reply:
Alright then.

Reply:
Please do that in its own commit though, separated from the move to org.kde.Platform//5.12, explaining the above.